migrations/versions/ce340b5b9d1e_added_thumbnail_image_column_to_project_.py

```diff
@@ -0,0 +1,44 @@
+"""Added thumbnail_image column to Project model
+
+Revision ID: ce340b5b9d1e
+Revises: 017c60414c03
+Create Date: 2023-10-13 19:38:30.010804
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = 'ce340b5b9d1e'
+down_revision: str = '017c60414c03'
+branch_labels: str | tuple[str, ...] | None = None
+depends_on: str | tuple[str, ...] | None = None
+
+
+def upgrade(engine_name: str = '') -> None:
+    """Upgrade all databases."""
+    # Do not modify. Edit `upgrade_` instead
+    globals().get(f'upgrade_{engine_name}', lambda: None)()
+
+
+def downgrade(engine_name: str = '') -> None:
+    """Downgrade all databases."""
+    # Do not modify. Edit `downgrade_` instead
+    globals().get(f'downgrade_{engine_name}', lambda: None)()
+
+
+def upgrade_() -> None:
+    """Upgrade default database."""
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Remove these lines. They're a reminder that you should review before committing.